features/user/account/account_password.feature

```diff
@@ -33,10 +33,14 @@ Feature: User account password change
 
     Scenario: Successfully changing my password
         Given I am on my account password page
-         When I fill in "Current password" with "sylius"
-          And I fill in "New password" with "newpassword"
-          And I fill in "Confirmation" with "newpassword"
-          And I press "Save changes"
+         When I change my account password to "newpassword"
          Then I should be on my account homepage
           And I should see "Your password has been changed successfully!"
 
+    Scenario: Logging in after password change
+        Given I am on my account password page
```

Comment:
Is this step necessary? IMO we could start with `Given I changed my account password to "newpassword"`